packages/sdk-demo/src/components/Menu/consts.tsx

```diff
@@ -46,6 +47,11 @@ export const getNavigationData = (
         url: ROUTES.approvalPolicies,
         renderIcon: (props) => <TabIcon {...props} />,
       },
+      approvalRequests: {
```

Comment:
`approvalRequests` must be in the main menu list, not in the `Settings`

<img width="236" alt="image" src="https://github.com/team-monite/monite-sdk/assets/725645/09da5873-a46c-499b-b34c-f649102081d8">

Reply:
I put it next to the Payables because they are related. done